shell/browser/native_window_mac.mm

```diff
@@ -1653,6 +1654,11 @@ void ViewDidMoveToSuperview(NSView* self, SEL _cmd) {
   return is_active_;
 }
 
+void NativeWindowMac::ReorderButtonsView() {
+  [buttons_view_ removeFromSuperview];
```

Comment:
Should check `if (buttons_view_)` here since `AddBrowserView` does not do the check.